cmd/influxd/launcher/cmd.go

```diff
@@ -141,10 +141,11 @@ type InfluxdOpts struct {
 	Testing                 bool
 	TestingAlwaysAllowSetup bool
 
-	LogLevel          zapcore.Level
-	FluxLogEnabled    bool
-	TracingType       string
-	ReportingDisabled bool
+	LogLevel           zapcore.Level
+	FluxLogEnabled     bool
+	LogShutdownQueries bool
```

Comment:
I don't think we should have a new config value for this - just logging on shutdown whenever FluxLogEnabled is true should be sufficient? @DStrand1 are you able to pick this one up now that @wbaker85 is on the compute team?